packages/@coorpacademy-app-review/src/test/index.test.tsx

```diff
@@ -4,10 +4,10 @@ import test from 'ava';
 import type {ExecutionContext} from 'ava';
 import React from 'react';
 import {render, fireEvent, act} from '@testing-library/react';
+import {services} from '@coorpacademy/review-services-mocks';
+import {sleep} from '@coorpacademy/review-services/es/tools/sleep';
```

Comment:
`sleep` should not be exported by the `review-services` lib (it's not a service). Please check if this function is used internally by the `review-services`. If not, the function must be deleted there and defined here (for internal use of `app-review`).